tests/solidity_test_helpers/MockDisputeBondToken.sol

```diff
@@ -57,6 +57,14 @@ contract MockDisputeBondToken is ITyped, IDisputeBond {
         return _market.decreaseExtraDisputeBondRemainingToBePaidOut(_amount);
     }
 
+    function callMigrateOutDisputeBondToken(IReputationToken _reputationToken, IReputationToken  _destination, address _reporter, uint256 _attotokens) public returns(bool) {
+        return _reputationToken.migrateOutDisputeBondToken(_destination, _reporter, _attotokens);
+    }
+
+    function callMintForDisputeBondMigration(IReputationToken _reputationToken, uint256 _attotokens) public returns(bool) {
+        return _reputationToken.mintForDisputeBondMigration(_attotokens);
+    }
```

Comment:
@nuevoalex has a PR out that makes `DisputeBond` not a token.  Depending on who merges first, this may need to be deleted.

Reply:
The race is on!!!